r3bsource/neuland/R3BNeulandTamexReader.cxx

```diff
@@ -157,16 +157,16 @@ Bool_t R3BNeulandTamexReader::R3BRead()
     }
 
     // Cards' trigger.
-    if (fArrayTrigger)
+    if (fArrayTrigger != nullptr)
     {
         int v_i = 0;
-        for (int i_i = 0; i_i < fData->NN_TRIGCLM; ++i_i)
+        for (int i_i = 0; i_i < fData->NN_TRIGCM; ++i_i)
         {
-            auto ch = fData->NN_TRIGCLMI[i_i];
-            for (; v_i < fData->NN_TRIGCLME[i_i]; ++v_i)
+            auto ch = fData->NN_TRIGCMI[i_i];
+            for (; v_i < fData->NN_TRIGCME[i_i]; ++v_i)
             {
-                auto coarse = fData->NN_TRIGCLv[v_i];
-                auto fine = fData->NN_TRIGFLv[v_i];
+                auto coarse = fData->NN_TRIGCv[v_i];
+                auto fine = fData->NN_TRIGFv[v_i];
                 auto mapped = new ((*fArrayTrigger)[fArrayTrigger->GetEntriesFast()]) R3BPaddleTamexMappedData(0, ch);
                 mapped->fCoarseTime1LE = coarse;
                 mapped->fFineTime1LE = fine;
```

Comment:
This warning is caused by the use of indices, whose values could be larger than the size of the array. It's considered to be unsafe if we follow a stricter rule. To fix this, we should use something like `std::span` which restricts the out-of-bound reading. But I guess this is not the purpose of this PR. So let's first get over with it by disabling this warning with `NOLINTBEGIN` and `NOLINTEND`:
```cpp
// NOLINTBEGIN
if (fArrayTrigger != nullptr)
    {
        int v_i = 0;
        for (int i_i = 0; i_i < fData->NN_TRIGCM; ++i_i)
        {
            auto ch = fData->NN_TRIGCMI[i_i];
            for (; v_i < fData->NN_TRIGCME[i_i]; ++v_i)
            {
                auto coarse = fData->NN_TRIGCv[v_i];
                auto fine = fData->NN_TRIGFv[v_i];
                auto mapped = new ((*fArrayTrigger)[fArrayTrigger->GetEntriesFast()]) R3BPaddleTamexMappedData(0, ch);
                mapped->fCoarseTime1LE = coarse;
                mapped->fFineTime1LE = fine;
            }
        }
    }
// NOLINTEND
```